src/App.tsx

```diff
@@ -1,35 +1,28 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import {RouterProvider, createBrowserRouter} from "react-router-dom";
+import StadiumLayout from "./components/Stadium/Layout";
+import MatchLayout from "./components/Match/Container";
+import RootLayout from "./pages/Root";
+import Home from "./pages/Home";
+import ErrorPage from "./pages/Error";
+const router = createBrowserRouter([
+    {
+        path: "/",
+        element: <RootLayout />,
+        errorElement: <ErrorPage />,
+        children: [
+            {path: "/", element: <Home />},
+            {path: "/stadium", element: <StadiumLayout />},
```

Comment:
Path also better to move to a separate file like this:
export const HOME = "/";
export const STADIUM_ROUTE = "/stadium";
....
and just import it here
It`ll help to easier find all routes and change them if needed